app/src/main/java/com/android/voyageur/ui/components/PlaceSearchWidget.kt

```diff
@@ -20,22 +25,41 @@ import com.android.voyageur.model.place.PlacesViewModel
 fun PlaceSearchWidget(
     placesViewModel: PlacesViewModel,
     onSelect: (CustomPlace) -> Unit,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    query: TextFieldValue,
+    onQueryChange: (TextFieldValue) -> Unit
```

Comment:
Do we really need the query to be exported in the classes that use the search widget? I think the purpose of this widget was to be ready out of the box. Simply clicking on a dropdown item was going to perform the OnSelect function on that specific place.

Reply:
However, we could leave it like this if it is absolutely necessary

Reply:
Just using the initial widget didn't work, so I had to add it as a parameter. Maybe for the next sprint we can find a workaround, but it is not a necessity.

Reply:
For now it is fine, but we should look into it in order to make the code more modular. As this is not a very good practice :)